com.github.edenalencar.identifications.json

```diff
@@ -0,0 +1,44 @@
+{
+    "app-id" : "com.github.edenalencar.identifications",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "identifications",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "identifications",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/edenalencar/identifications.git",
+                    "commit": "364094021551fcc3094cf828dd7fb27fe63f4f8d",
+                    "tag": "0.1.1",
```

Comment:
Appdata (upstream) still say 0.1.0. So the version displayed will be inconsistent.

Reply:
Solved.